guides/schematics.md

```diff
@@ -0,0 +1,53 @@
+Angular Material comes packaged with Angular CLI schematics to make
+creating new components easy.
```

Comment:
Does this make creating Material applications easier? Or just components?

Reply:
Oh good point, I think it should be applications.